### ISSUE TITLE
Would you be willing to add a license file?

### ISSUE THREAD

**Issue by bannanc** [ehermes/ased3#2]
Hello, I've been working on trying to use [ASE_ANI](https://github.com/isayev/ASE_ANI) from @isayev which recommends adding ased3. However, without a license file it isn't clear if we are allowed to download and use your package. Would you be willing to add a license? Many academic groups producing code use the [MIT license](https://opensource.org/licenses/MIT) which allows for user to use and modify the code base as long as they cite the original creators. 

Thanks!

**Comment by ehermes** [comment]
I've added my preferred license, LGPLv3. I believe ASE_ANI's usage of my software is compatible with this license.

As an aside, I suggest you do not use ased3 (EDIT: original wording made it sound like I was suggesting not to use ASE_ANI, which was not the intent). I also wrote [ASE's built-in DFT-D3 calculator](https://wiki.fysik.dtu.dk/ase/ase/calculators/dftd3.html), which uses the reference implementation of DFT-D3. It isn't parallel, but the parallelism in this repository is pretty dang buggy!
